### DISCUSSION TITLE
Does HarfBuzz support Deprecated Format Characters U+206A–U+206F

### DISCUSSION THREAD

**Discussion by Eli-Zaretskii** [harfbuzz/harfbuzz#3472] [Q&A] [answered]
A question: Does HarfBuzz support the U+206C inhibit arabic form shaping and U+206D activate arabic form shaping formatting controls?  I know their use is strongly deprecated by Unicode, but are they supported as described in the Unicode Standard, i.e. do they inhibit and activate the shaping of Arabic Presentation Forms?

Same question about U+206A inhibit symmetric swapping and U+206B activate symmetric swapping.

Thanks in advance.

**Comment by behdad** [comment, marked as answer]
We don't.

**Comment by Eli-Zaretskii** [comment]
Thanks.
